mvrx-rxjava3/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest package="com.airbnb.mvrx.rxjava2">
```

Comment:
i don't think this is needed. declare the package name in the build.gradle file via the `namespace` property